stories/Button.stories.ts

```diff
@@ -11,7 +11,7 @@ import { IconOptions } from './Icon.stories';
 
 // More on how to set up stories at: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
 const meta: Meta<typeof Button> = {
-  title: 'Components/Button',
+  title: 'Components/Components/Button',
```

Comment:
Why do create another Components folder for the storybook?

Reply:
no special idea here. we can also keep them on the root.  I thought to make a separation between base components like typo, icons. for me no problem at all to revert this.